tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationView.java

```diff
@@ -606,6 +610,21 @@ private void updateForm( SelectionChangedEvent event )
         }
     };
 
+    private void addWorkspaceResourceListener()
```

Comment:
This will get called multiple times which means there will be multiple resource listeners added.  Can you change this so only one resource listener per eclipse instance is added.